prombench/docs/eks.md

```diff
@@ -0,0 +1,111 @@
+# Prombench in EKS
+
+Run prombench tests in [Elastic Kubernetes Service](https://aws.amazon.com/eks/).
+
+## Setup prombench
+
+1. [Create the main node](#create-the-main-node)
+2. [Deploy monitoring components](#deploy-monitoring-components)
+
+### Create the Main Node
+
+---
+
+- Create [security credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) on AWS. Put the credentials into a `yaml` file as follows:
+```yaml
+accesskeyid: <Amazon access key>
+secretaccesskey: <Amazon access secret>
+```
+- Create a [VPC](https://docs.aws.amazon.com/eks/latest/userguide/create-public-private-vpc.html) with public subnets.
+- Create a [Amazon EKS cluster role](https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html) with following policies:
+    - AmazonEKSclusterPolicy 
+- Create a [Amazon EKS worker node role](https://docs.aws.amazon.com/eks/latest/userguide/worker_node_IAM_role.html) with following policies:
```

Comment:
Very small nit (feel free to ignore this one), you can probably just do `AmazonEKSWorkerNodePolicy`, `AmazonEKS_CNI_Policy`, `AmazonEC2ContainerRegistryReadOnly`